components/collapsible/collapsible_lazy_show.vue

```diff
@@ -67,6 +68,19 @@ export default {
     };
   },
 
+  /******************
+   *    COMPUTED    *
+   ******************/
+  computed: {
+    /**
+     * Set the css property to false when running tests only.
+     * @returns {boolean}
+     */
+    isCSSEnabled () {
+      return process.env.NODE_ENV !== 'test';
+    },
+  },
+
```

Comment:
Kinda awkward but guess it works :shrug:. Maybe add a link to this page to explain what the css property does https://vuejs.org/guide/built-ins/transition.html#javascript-hooks